config/console.php

```diff
@@ -6,6 +6,9 @@
 
 return [
     Aliases::class => [
-        '@yiisoft/yii/db/migration' => dirname(__DIR__)
+        '__class' => Aliases::class,
+        '__construct()' => [
```

Comment:
That's more explicit but previous code should work as well.

Reply:
yes, its correct close pr.